Tests/DuplicatedSDKTest/DuplicatedSDKTest/DuplicatedSDKTest-Bridging-Header.h

```diff
@@ -0,0 +1,14 @@
+
+#import <Sentry/Sentry.h>
```

Comment:
`m`: I would prefer moving this project to `TestSamples/`.

Reply:
I would prefer to move TestSamples to be a sub folder Tests. Actually Im thinking about doing that in a another PR.

Reply:
Also fine. I don't want two different location for the same thing 👍